CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 IMPROVEMENTS:
  * agent: allow the job GC interval to be configured [[GH-5978](https://github.com/hashicorp/nomad/issues/5978)]
+ * api: add follow parameter to file streaming endpoint to support older browsers [[GH-6049](https://github.com/hashicorp/nomad/pull/6049]
```

Comment:
i think this is missing a closing parenthesis
these are easy to mess up by copy-pasting. we tend to use `[GH-####]` and then run `make changelogfmt`

Reply:
Ah, thanks for catching that. Done.

I'll merge this once CI gives the :+1: again.